RELEASE_NOTES.md

```diff
@@ -2,6 +2,10 @@
 
 # Unreleased
 
+## Platform Additions
+
+Enable macOS-10.14 (Mojave) tested packages.
```

Comment:
Can we just say:

"macOS 10.14 (Mojave) is now fully tested and packages are available on downloads.chef.io."